kie-dmn-core/src/test/java/org/kie/dmn/core/DMNRuntimeTest.java

```diff
@@ -733,7 +733,7 @@ public void testInvalidVariableNames() {
         DMNModel dmnModel = runtime.getModel( "https://github.com/droolsjbpm/kie-dmn", "0001-input-data-string" );
         assertThat( dmnModel, notNullValue() );
         assertThat( dmnModel.hasErrors(), is(true) );
-        assertThat( formatMessages( dmnModel.getMessages() ), dmnModel.getMessages().size(), is(4) );
+        assertThat( formatMessages( dmnModel.getMessages() ), dmnModel.getMessages().size(), is(10) );
```

Comment:
`Person)`
`not ...`
`some ...`
`my ^ ...`
`Foo # ...`
additionally `message='Error compiling FEEL expression 'some number' for name 'my ^ function' on node 'Monkeys in the jungle':`